python_modules/dagster/dagster/core/asset_defs/asset_group.py

```diff
@@ -260,9 +257,34 @@ def build_job(
                 executor_def=executor_def,
                 description=description,
                 tags=tags,
+                _asset_group=self,
             )
         return asset_job
 
+    def build_asset_selection_job(
```

Comment:
do we actually need this as a separate method vs just calling build_job()? The only difference seems to be that we'd need to translate asset_selection List[AssetKey] into selection List[str]. Might be missing something